docs/.sphinx/spellingcheck.yaml

```diff
@@ -9,7 +9,13 @@ matrix:
     - .custom_wordlist.txt
     output: .sphinx/.wordlist.dic
   sources:
-  - _build/**/*.html
+      # check all files except dconf type in the technical reference
+  - _build/tutorial/**/*.html
+  - _build/explanation/**/*.html
+  - _build/how-to/**/*.html
+  - _build/reference/policies/Computer Policies/Ubuntu/Client management/Computer Scripts/**/*.html|!_build/reference/policies/Computer Policies/Ubuntu/Client management/Power Management/**/*.html
+  - _build/reference/policies/User Policies/Ubuntu/Session management/**/*.html
+  - _build/reference/policies/User Policies/Ubuntu/Peripherals/**/*.html
```

Comment:
Thanks @jibel -- your one-liner was what I originally tested locally to resolve the spelling errors.
I changed it so that specific files in `/policies/` (that we control) could be included; that was something specified in the Jira card and that I originally tried to resolve with regex (not pretty!).

I agree that the one-liner alternative would be easier to maintain and maybe this is a compelling reason to use it.

Could you clarify how/if we want to capture the Scripts, Session and Peripherals files?
If I read you correctly, you are suggesting we just ignore those too for now??
I wouldn't have a problem with that, especially if there is a maintainability argument.
I also don't expect there to be show-stopping spelling errors going from the .go and .yaml files in the near future.

Reply:
Right, I suggest that we ignore those.
These strings come from upstream (from schemas in `/usr/share/glib-2.0/schemas/`) and if we want to track spelling mistakes it should be tracked at the source. adsys is just a consumer and there isn't much we can do at this level.